src/main/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+spring.web.resources.add-mappings=true
+spring.profiles.active=dev
+
+spring.datasource.url = ?
+spring.datasource.driver-class-name = ?
+spring.datasource.username=name
+spring.datasource.password=password
+
+
+spring.servlet.multipart.max-file-size=720B
+spring.servlet.multipart.max-request-size=720KB
```

Comment:
I did attempt to fix the  error when running the program. When pulling the project there was no application.Properties so I added it since most of the feedback I was getting involved this part based off what I could gather..